src/pages/Systems/MachineInfo.tsx

```diff
@@ -98,16 +107,24 @@ const IngestorInfo = () => {
 						</ThemeIcon>
 					</Tooltip>
 				)}
+				{!ingestor?.reachable ? (
+					<Stack style={{ width: '85%' }}>
+						<Group justify="flex-end">
+							<Button onClick={() => setOpened(true)} className={classes.deleteIcon} color="white">
+								<IconTrash height="18" stroke={1.5} />
```

Comment:
use rem for sizes

Reply:
<IconTrash> height prop doesnt allow css units it takes only number as input

Reply:
use size like everywhere else.